packages/ethereum-contracts/utils/mfa-tester.html

```diff
@@ -0,0 +1,631 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Multi Flow SuperApp Tester</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous" />
+    <script src="https://cdn.jsdelivr.net/npm/ethers@5.6.0/dist/ethers.umd.min.js"
+        integrity="sha256-I/T7C/qM1p/EIawMnVp+b68aGIgHpiaW5uJm8Xd/WYg=" crossorigin="anonymous"></script>
+
+    <!-- sdk-core -->
+    <script src="./index.umd.js"></script>
```

Comment:
maybe create a new level of subfolder "dist", which is automatically git-ignored